.github/workflows/tag.yml

```diff
@@ -58,15 +59,46 @@ jobs:
             mondoo/client:${{ steps.semver.outputs.major }}
             mondoo/client:latest
 
-      - name: Build and push (ubi)
+      - name: Build and push rootless images (alpine)
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/386,linux/amd64,linux/arm64,linux/arm/v6,linux/arm/v7
+          push: true
+          build-args: VERSION=${{ steps.vars.outputs.tag }}
+          target: rootless
+          tags: |
+            mondoolabs/mondoo:${{ steps.vars.outputs.tag }}-rootless
```

Comment:
the rootless image should only be in the new namespace.

Reply:
done